source/localizable/getting-started/quick-start.md

```diff
@@ -243,6 +243,50 @@ create a `programmers` route that shows a list of famous programmers. By
 re-using the `people-list` component, you can do it in almost no code at
 all.
 
+## User Events
```

Comment:
I've also been watching a lot of TableTop, so the user doing an action is verbiage that sounds nice to me 👼